### DISCUSSION TITLE
malloc(): invalid size (unsorted) at E_System code

### DISCUSSION THREAD

**Discussion by dolgae03** [ANLAB-KAIST/KENSv3#127] [Project Q&A 2. Transport Layer (TCP)] [answered]
While testing reliable - transfer test case, The malloc error is occured out of my code. And i don't know why it is occured.

Is there any idea about it..???

![image](https://user-images.githubusercontent.com/109707681/204129963-e60eafa7-b390-4b2a-afea-e508e4b37824.png)

![image](https://user-images.githubusercontent.com/109707681/204129973-51d126a7-190a-4987-a133-e7ddc86633db.png)

![image](https://user-images.githubusercontent.com/109707681/204129985-3ca1b7c0-6938-4a8f-ae89-bfcd0b3ca836.png)

**Comment by whkim0** [comment, marked as answer]
As you know, memory bugs can be anywhere. Please check your code carefully, especially for `runnableReady` and `wake()`. I also recommend using sanitizer: https://github.com/ANLAB-KAIST/KENSv3/wiki/Tip:-Debugging-Memory-Bugs-(Only-for-macOS-and-Linux).

**Reply by dolgae03** [reply]
I solved this problem by fix my code.. Thank you
